tests/db/compose.yml

```diff
@@ -93,7 +93,9 @@ services:
       context: .
       dockerfile: Dockerfile.mssql
     environment:
-      MLFLOW_TRACKING_URI: mssql+pyodbc://mlflowuser:Mlfl*wpassword1@mssql/mlflowdb?driver=ODBC+Driver+17+for+SQL+Server
+      # `LongAsMax=Yes` is to fix error in sqlalchemy<2.0:
+      # [ODBC Driver 17 for SQL Server][SQL Server]The data types varchar and ntext are incompatible in the equal to operator
```

Comment:
Can we include https://docs.sqlalchemy.org/en/20/dialects/mssql.html#avoiding-sending-large-string-parameters-as-text-ntext in the comment?